.eslintrc.json

```diff
@@ -360,7 +361,8 @@
         "packages/hash/playwright/**"
       ],
       "rules": {
-        "@typescript-eslint/prefer-nullish-coalescing": ["off"]
+        "@typescript-eslint/prefer-nullish-coalescing": ["off"],
+        "github/array-foreach": ["off"]
```

Comment:
I assume this line is temporary, but if it is not, it’d be great to add some comment somewhere. Perhaps, we should finally rename `.json` to `.js` to allow comments. Doing so is probably out of PR scope, but not sure.